docker/boot/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apk add nginx && \
     mkdir -p /var/cache/nginx && \
     chown -R www:www /var/cache/nginx && \
     chown -R www:www /var/tmp/nginx && \
```

Comment:
I'm receiving an error during container build
chown: /var/tmp/nginx: No such file or directory
Please fix this